src/values/Value.js

```diff
@@ -34,6 +34,7 @@ export default class Value {
     switch (typeName) {
       case "empty": return EmptyValue;
       case "undefined": return UndefinedValue;
```

Comment:
Hm, interestingly, "undefined" already existed before, and then would have just crash? Can you remove "undefined"?